tests/rule_based_profiler/test_rule_based_profiler.py

```diff
@@ -948,6 +953,7 @@ def test_run_profiler_with_dynamic_args(
     assert mock_profiler_run.call_args == mock.call(
         variables=variables,
         rules=rules,
+        expectation_suite=None,
```

Comment:
would the default here not suffice?

Reply:
Unfortunately not - although the default is indeed `None`, `call_args` on a mock required very explicit assertions. Since this profiler run was invoked with this kwarg (albeit implicitly), we need to note it in our assertion.